domain/src/test/java/org/oppia/android/domain/platformparameter/syncup/PlatformParameterSyncUpWorkManagerInitializerTest.kt

```diff
@@ -119,7 +122,7 @@ class PlatformParameterSyncUpWorkManagerInitializerTest {
       PlatformParameterSyncUpWorker.PLATFORM_PARAMETER_WORKER
     ).build()
 
-    assertThat(syncUpWorkManagerInitializer.getSyncUpWorkRequestData()).isEqualTo(
+    assertThat(getSyncUpWorkRequestData()).isEqualTo(
       workerTypeForSyncingUpParameters
     )
```

Comment:
Can this fit on one line? If so, prefer to update (in general, try to keep everyone on one line if it fits vs. adding extra wrapping).